### ISSUE TITLE
cargo owner --add  error

### ISSUE THREAD

**Issue by foosaa** [Hirevo/alexandrie#156]
```bash
cargo owner -a abc --registry xxx
```
Error info：

```bash
    Updating `xxx` index
error: failed to invite owners to crate `crate-name` on registry at http://......

Caused by:
  invalid type: string "true", expected a boolean at line 1 column 56
```

I add a user to the crate, and the above error occurs, how can I solve it? Thanks！！！

**Comment by Hirevo** [comment]
I am indeed able to reproduced this, thanks for filing the issue.  

While an error indeed shows up on the command-line, your operation should have worked and the new owner should have been successfully added.  

You can try to run the following command to check if the operation has indeed still went through:
```bash
cargo owner --registry xxx abc --list
```

Or if you are using the frontend, you can check if the new owner is listed on the crate page for `abc`.

**Comment by foosaa** [comment]
When i execute：
```bash
cargo owner --registry xxx abc --list
```
only see myself

**Comment by foosaa** [comment]
@Hirevo 

Hi, Hirevo. Thank you for your hard work. I saw that the issue has been resolved after the update. 
To enter a specific crate directory, execute:
`cargo owner --add user@email.com --registry registryName`
Will receive the following prompt:

```bash
Updating `registryName` index
       Owner  has been added as authors of crateName
```

To view the owner list, use the command:
`cargo owner --list --registry registryName`
Will receive the following prompt:

```bash
Updating `registryName` index
user1@email.com (user1)
user2@email.com (user2)
```

After verification, the added owners can also publish the crate.

Thank you once again, Hirevo.
